library/src/main/java/com/wdullaer/materialdatetimepicker/date/DatePickerDialog.java

```diff
@@ -576,6 +585,13 @@ public void showYearPickerFirst(boolean yearPicker) {
         mDefaultView = yearPicker ? YEAR_VIEW : MONTH_AND_DAY_VIEW;
     }
 
+    public void showYearOnly(boolean yearOnly) {
+        mShowYearOnly = yearOnly;
+        if (yearOnly) {
```

Comment:
This will fail if someone later set showYearPickerFirst to false
You'll have to change the setter for showYearPickerFirst too
```java
mShowYearFirst = yearOnly || showYearFirst;
```
You'll also have to add some logic to the sample app that takes this paired logic into account. (would be great if android had some nice way to do reactive UIs, but there you go)